pyth-sdk-terra/README.md

```diff
@@ -0,0 +1,71 @@
+# Pyth SDK Solana
+
+This crate provides utilities for reading price feeds from the [pyth.network](https://pyth.network/) oracle on the Terra network.
+The crate includes a library for reading and using Pyth data feeds in Terra.
+
+## Usage
+
+### Read price
+ 
+For reading the price you just need to call `query_price` function within your contract.
+
+```rust
+// let price_id = Binary::from(b"xyz...");
+
+// Contract address is defaulted to the mainnet address, for testnet use `testnet` feature flag.
+let pyth_contract_addr = pyth_sdk_terra::CONTRACT_ADDR;
+
+let price: Price = query_price_info(deps.querier, contract_addr, price_id).unwrap().price;
```

Comment:
So this is how things were in pyth-client-rs, which was just as confusing there as here. I would be hugely in favor of renaming this type. `PriceFeed` is a bit of a misnomer because it implies you could read more prices from the type than just the current but the current API doesn't define this right now but this is still much better than the previous naming is.

This would also let us rename `PriceConf` to just `Price` which seems like it might be the better name for that type also. Thoughts @ali-bahjati ?

Reply:
Done.